### PR TITLE
[Remove Stream/Optional usage] LPS-178728 headless-commerce-admin-order-impl

### PR DESCRIPTION
**Pull request by JanisZhang**
https://issues.liferay.com/browse/LPS-178728

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:stop

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:commerce

**Comment by catarineaguiar** [comment]
ci:stop

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/resource/v1_0/OrderResourceImpl.java [hunk 1]
```diff
@@ -495,53 +492,49 @@ private Method _getMethod(Class<?> clazz, String methodName) {
 	private String[] _getOrderItemExternalReferenceCodes(
 		OrderItem[] orderItems) {
 
-		Stream<OrderItem> stream = Arrays.stream(orderItems);
+		Set<String> strings = new HashSet<>();
```
Comment (dantewang) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/resource/v1_0/OrderResourceImpl.java:495]: @JanisZhang Please make these variable names match the method names.

**Pull request closed without merging**